code/__DEFINES/sound.dm

```diff
@@ -1,3 +1,4 @@
 //max channel is 1024. Only go lower from here, because byond tends to pick the first availiable channel to play sounds on
 #define CHANNEL_LOBBYMUSIC 1024
 #define CHANNEL_ADMIN 1023
+#define CHANNEL_VOX 1022
```

Comment:
hey can you make a proc that just does ```rand(1, LOWEST_ONE_OF_THESE_WE_HAVE)``` and have the sound channel be:
```S.channel = channel || openSoundChannel()```
yes its only a 1/1024 chance of hitting one of these, but we can prevent it so why don't we!